consumer/hikey960/installation/linux-fastboot.md

```diff
@@ -88,13 +88,34 @@ If your HiKey960 is pre-flashed with appropriate firmware, you should see your d
 
 ## Step 5: Flash Operating System
 
+###### AOSP
```

Comment:
Sub heading greyed out. It might be better to use ####?

Reply:
ok will use ####.